.travis.yml

```diff
@@ -11,19 +9,17 @@ cache:
     - $HOME/.sbt/boot/
 
 script:
-  - /bin/true
+  - ./scripts/cibuild
 
 before_deploy:
-  - mkdir -p ${HOME}/.sbt
-  - docker run -v ${HOME}/.ivy2:/root/.ivy2 -v ${HOME}/.sbt:/root/.sbt -v ${PWD}:/mmw-geoprocessing -w /mmw-geoprocessing quay.io/azavea/scala:2.10.5 ./sbt assembly
-  - sudo chown ${USER} summary/target/scala-2.10/mmw-geoprocessing-assembly-${TRAVIS_TAG}.jar
+  - sudo chown ${USER} api/target/scala-2.11/mmw-geoprocessing-api-assembly-#{TRAVIS_TAG}.jar
```

Comment:
The `sudo chown` stuff here was to get around Docker volume permission issues. If we avoid Docker, we can also drop this too.